config.lua.dist

```diff
@@ -46,6 +46,7 @@ loginProtocolPort = 7171
 gameProtocolPort = 7172
 statusProtocolPort = 7171
 maxPlayers = 0
+worldId = 1
```

Comment:
I think this shouldn't be in the config. Register the world by name and let auto-increment give you and ID. You can then query the database for this to use the key everywhere else.

Name should be unique, and indexed, so should be simple to do. Having the ID in the config is leaking the database abstractions and makes it potentially a bit annoying to get right.